### ISSUE TITLE
Can automatic highlight certain type, CP or IV pokemon?

### ISSUE THREAD

**Issue by edmondhk** [Wolfsblvt/BlossomsPokemonGoManager#123]
It would be nice if it can automatic highlight certain type, CP or IV pokemon?
Even better with different colours.
High CP in red
High IV in yellow
Mewtwo in green

Of course it have to define in option.

**Comment by QuAzI** [comment]
It will be great if types colored as defined in game or like this
http://www.fanboysanonymous.com/2016/02/explaining-why-pokemon-type-advantages.html

Some IV calculators color IV column: low level - red background, than pink, yellow, white for normal pokemons, lime and green for best pokemons. It helps choose better pokemon for evolve.

Example of IV colors
![_2016-08-09_16 35 48_16 35rrrrrrr](https://cloud.githubusercontent.com/assets/1897605/17518184/82f18c78-5e4f-11e6-92e2-3b46aaf25d7a.png)

**Comment by QuAzI** [comment]
I think pokemons which can't be evolved must have gray columns "Candies" and "To evolve"

**Comment by Wolfsblvt** [comment]
Color highlighting would be a really cool thing. Will be planned later with the new UI.

**Comment by QuAzI** [comment]
If pokemon in GYM it cannot be transferred, evolved or upgraded. This row can be grayIf pokemon in GYM it cannot be transferred, evolved or upgraded. This row can be completely gray (like inactive control)

**Comment by Wolfsblvt** [comment]
Planned ToDos on that from here:
- [ ] Color-code ascending/descending columns (if wanted), like CP and IV
- [ ] Grey out rows with gym pokemons that can't be used right now
- [ ] Grey out "To Evolve" if it can't be evolved

That means it contains some requests from #44.

**Comment by ritsu** [comment]
I was working on something similar before I decided to drop it and use this wonderful app. Here's a screenshot of what I had when I stopped, in case it helps give any ideas.

![pokemon](https://cloud.githubusercontent.com/assets/490097/17615269/169b296a-603b-11e6-9482-73db6ea2ff1b.png)

I find pokemon icons easier to read than pokemon names, but it might not be a good idea to include them here because of copyright issues. Maybe have an option to allow the user to use icons they've downloaded elsewhere? (e.g. https://github.com/Superviral/Pokemon-GO-App-Assets-and-Images) Or is that still too complicit / not worth the effort?

**Comment by Wolfsblvt** [comment]
Yeah, most of things are already planned and will be added with the new GUI (: Like IV coloring, Favorite stars, that gym thingy.

Sprites are an idea and maybe possible, yes. Was noted already in #49.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using BlossomsPokemonGoManager. This project is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving it. Please do not use it with a live account or current credentials. See the [Disclaimer](https://github.com/Wolfsblvt/BlossomsPokemonGoManager#disclaimer) for details.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. Blossoms Pokémon Go Manager is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.
